### PR TITLE
chore: fail asciidoctor if there are warnings

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
@sterobin @bhardesty This one is there for a while. Number of scripts were updated including modular docs. 
They are working locally. If you can take look into this

**Comment by sterobin** [comment]
@wtrocki Sorry, didn't know you were waiting for approval from us since it was under-the-hood updates. Sounds good. Merging.

**Pull request merged**
